apps/ai_reports/models.py

```diff
@@ -9,6 +9,7 @@ class AiReport(TimeStampedModel):
     category = models.CharField(max_length=50)
     explanation = models.TextField()
     confidence = models.FloatField(default=0)
+    show_ai_report = models.BooleanField(default=True)
```

Comment:
I would rename `show_ai_report` to `show` or `is_visible` because it is a method of `ai_report` and I wouldn't repeat things if not necessary

Reply:
yes that's the principle. however here is a particular case, since we are looking for this field to update the comment in async js (toggling the button), and looking for "show/visible" would not point we are interested in the ai report, rather than the user's report.
ref:
https://github.com/liqd/a4-defakts/pull/32/files#diff-b079089215497b5cee99b986d535f03b6972fca4c3801a62cda935f840833693R89

https://github.com/liqd/a4-defakts/pull/32/files#diff-32233bec0fed3e4c5f0b60ae8cc74517e9a6c083cad0582e0c3329f525a51088R223

Reply:
@hklarner  @m4ra  I guess we could keep the `show_ai_report` in the request / action and only change it here on the model if we want to. I'm fine with both options though, so can go with whatever you prefer.

Reply:
renamed to `show_in_discussion`